backend/carbon-services-lib/src/data-importer/importers/itmo-system.service.ts

```diff
@@ -223,18 +223,18 @@ export class ITMOSystemImporter implements ImporterInterface {
                   }))
               
               this.logger.log(`Issue list values: ${list}`)
-              if (list && list.includes("Upload Final Monitoring Report")) {
```

Comment:
Why did you comment this part?

Reply:
Now the requirement is to issue credits through mitigation action. In the ITMO system, there is no mitigation actions option. Since the development of ITMO system is ongoing, that part was commented out after discussion with Sapumal and Sewmini.